### ISSUE TITLE
Timelapse export mode

### ISSUE THREAD

**Issue by masc4ii** [ilia3101/MLV-App#111]
Can we make Time-Lapse export mode?

I shot MLV video @ 6 FPS and I don't want to render and process all the frames which are 6895 and size of 29.5 GB then speed it up in editing software I want to export 1 Frame after every 25 Frames for example or 1 Frame after 50 Frames this is changeable choice depending on the Time-Lapse speed you want but you got the Idea.

**Comment by bouncyball-git** [comment]
Would be cool. Then you have to increment the frame like you need. I guess it's  gonna be easily doable. Or am I wrong?

**Comment by DeafEyeJedi** [comment]
I dig this idea big time...

**Comment by masc4ii** [comment]
Right... we just need a variable which is accessable from export settings - something like "Timelapse skip frames [xx frames]". And in export we increment the frames with i=i+variable instead of i++.

**Comment by bouncyball-git** [comment]
I think this variable should affect playback too. For previewing whats the result gonna be.

**Comment by bouncyball-git** [comment]
@DeafEyeJedi 

Add I dig that you are still around 😃 

HAYD man?

**Comment by masc4ii** [comment]
Yeah... think so too... but that's difficult! :-/

**Comment by masc4ii** [comment]
There are more idea's by a1ex for this topic: "180-degree averaging, instead of skipping". But because of performance reasons, this would be export only then:
https://www.magiclantern.fm/forum/index.php?topic=20025.msg206662#msg206662

**Comment by bouncyball-git** [comment]
We can do both, skipping or averaging.
